app/uk/gov/hmrc/helptosavefrontend/repo/EnrolmentStore.scala

```diff
@@ -70,34 +73,59 @@ class MongoEnrolmentStore @Inject()(mongo: ReactiveMongoComponent)(implicit ec:
     )
   )
 
-  private[enrolment] def doUpdate(data: EnrolmentData)(implicit ec: ExecutionContext): Future[Option[EnrolmentData]] =
+  private[repo] def doCreate(nino: NINO, itmpFlag: Boolean, email: String)(implicit ec: ExecutionContext): Future[Option[EnrolmentData]] =
     collection.findAndUpdate(
-      BSONDocument("nino" -> data.nino),
-      BSONDocument("$set" -> BSONDocument("itmpHtSFlag" -> data.itmpHtSFlag)),
+      BSONDocument("nino" -> nino),
+      BSONDocument("$set" -> BSONDocument("itmpHtSFlag" -> itmpFlag, "email" -> encrypt(email))),
+      fetchNewObject = true,
+      upsert = true
```

Comment:
do we want to `upsert` in this case? We're creating, we shouldn't be updating existing entries